apollo-portal/src/main/resources/static/index.html

```diff
@@ -36,37 +36,221 @@
+                     ng-click="getUserFavorites()">
+                    <div href="#" class="thumbnail hover cursor-pointer"
+                         style="display: flex;justify-content: center;align-items: center">
+                        <div><img class="more-img" src="img/more.png" /></div>
+                        <div style="margin-left: 5px"><h5>{{'Index.LoadMore' | translate }}</h5></div>
+                    </div>
+                </div>
+            </div>
+            <div class="public-namespace-list" ng-show="whichContent==='3'">
+                <div class="display-table" ng-show="publicNamespaces.length > 0">
+                    <table>
+                        <tr>
+                            <th style="width: 20%">{{'Index.appTable.appId' | translate }}</th>
+                            <th style="width: 30%">{{'Index.appTable.nameSpace' | translate }}</th>
+                            <th style="width: 20%">{{'Index.appTable.CreatedTime' | translate }}</th>
+                            <th style="width: 30%">{{'Index.appTable.LastModifiedTime' | translate }}</th>
+                        </tr>
+                        <tr ng-repeat="app in publicNamespaces" ng-click="goToAppHomePage(app.appId)"
+                            href="#" class="hover cursor-pointer">
+                            <td style="width: 20%">{{ app.appId }}</td>
+                            <td style="width: 30%">{{ app.name }}</td>
+                            <td style="width: 20%">{{ app.dataChangeCreatedTime.slice(0,10) }}</td>
+                            <td style="width: 30%">{{ app.dataChangeLastModifiedTime.slice(0,10) }}</td>
+                        </tr>
+                    </table>
+                </div>
+                <div class="no-public text-center" ng-show="!publicNamespaces || publicNamespaces.length == 0">
+                    <h4>{{'Index.FavoriteTip' | translate }}</h4>
+                </div>
+                <div ng-show="!hasMorePublicNamespaces" style="height: 15px"></div>
+                <div style="width: 180px;margin-left: 30px;margin-top: 20px" ng-show="hasMorePublicNamespaces"
+                     ng-click="morePublicNamespace()">
+                    <div href="#" class="thumbnail hover cursor-pointer"
+                         style="display: flex;justify-content: center;align-items: center">
+                        <div><img class="more-img" src="img/more.png" /></div>
+                        <div style="margin-left: 5px"><h5>{{'Index.LoadMore' | translate }}</h5></div>
+                    </div>
+                </div>
+            </div>
+            <div class="visit-app-list" ng-show="whichContent==='4'">
+                <div class="display-table" ng-show="visitedApps.length > 0">
+                    <table>
+                        <tr>
+                            <th style="width: 20%">{{'Index.appTable.appId' | translate }}</th>
```

Comment:
The same as above, it's better to display the information as those in the project page: App Id, App Name, Department, App Owner, Email.